resources/js/Components/ContentRenderer/ContentRenderer.css

```
@@ -0,0 +1,327 @@
+    margin-bottom: 0.25rem;
+    line-height: 1.5;
+}
+
+.markdown-content blockquote {
+    border-left: 4px solid #3b82f6;
+    padding-left: 1rem;
+    margin: 1.5rem 0;
+    font-style: italic;
+    color: #4b5563;
+    background-color: #f8fafc;
+    padding: 1rem;
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)